ooasp/smart_ooasp.py

```diff
@@ -54,8 +54,7 @@ def __init__(
         self.view = view
 
         self.next_id = 1
-        self.associations = []
-        self.object_atoms = []
+        self.assumptions = set()
```

Comment:
Why a set?

Reply:
This was a change done in order to avoid duplicit entries that caused issues when interacting.

Reply:
@susuhahnml would know more details.

Reply:
Ok, makes sense!